contracts/dnr/dnr_test.go

```diff
@@ -16,6 +16,9 @@ import (
 )
 
 var ether = big.NewInt(1000000000000000000)
+var ren = big.NewInt(1000000000000000000)
```

Comment:
Can we call this `decimalMultiplier` instead of `ren`. It's more true to its actual usage.

Reply:
yes, changed it.